skycoin/skywire
# Add display node ip field to the main config

Did you run `make format && make check`?

Fixes [#skywire-services-610](https://github.com/SkycoinPro/skywire-services/issues/610)	

 Changes:	
- Added field `display_node_ip` to main config
- Removed field `display_node_ip` from private config
- Added `display_node_ip` to Services of service discovery

How to test this PR:
1. Checkout [#skycoin-service-discovery-118](https://github.com/SkycoinPro/skycoin-service-discovery/pull/118)
2. In service-discovery uncomment `//replace github.com/skycoin/skywire => ../skywire` in `go.mod` and run `make dep`
3. In `skywire-services` build integration env
4. Add `"display_node_ip": true` inside `"launcher"` of the config of visorB and also set `"is_public": true`
5. Check `http://service-discovery:9098/api/services?type=visor` and see if the IP of visorB shows not not.
